timer.js

```diff
@@ -0,0 +1,21 @@
+var timer = 180;
+
+function sleep(milliseconds) {
+    var start = new Date().getTime();
+    for (var i = 0; i < 1e7; i++) {
+      if ((new Date().getTime() - start) > milliseconds){
+        break;
+      }
+    }
+}
```

Comment:
Uhm. You definitely do not want a function like this here.

If you think you need this, use `setTimeout` or `setInterval`.

Reply:
the function is in place of setTimeout or setInterval because that only delays the start of your code and requires you to place brackets around each section that you want to be delayed. This alternative is much simpler and doesn't require you to put brackets around the part you want delayed.

Reply:
This function will cause your CPU to spin and the main thread to block. Nothing will be processed whilst this function runs (i.e. your dashboard will never update).